src/main/java/com/epam/lowcost/DAO/FlightDAOImpl.java

```diff
@@ -0,0 +1,148 @@
+package com.epam.lowcost.DAO;
+
+import com.epam.lowcost.model.Flight;
+import com.epam.lowcost.model.Plane;
+import com.epam.lowcost.util.DateFormatter;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FlightDAOImpl implements FlightDAO {
+    private DataSource dataSource;
+    private PlaneDAO planeDAO;
+
+    public FlightDAOImpl(DataSource dataSource, PlaneDAO planeDAO) {
+        this.dataSource = dataSource;
+        this.planeDAO = planeDAO;
+    }
+
+
+    @Override
+    public List<Flight> getAllFlights() {
+        List<Flight> flights = new ArrayList<>();
+        try (Connection conn = dataSource.getConnection();
+             Statement stmt = conn.createStatement();
+             ResultSet rs = stmt.executeQuery("SELECT * FROM FLIGHTS  ")) {
+            while (rs.next()) {
+                Long id = rs.getLong("id");
+                Long price = rs.getLong("initialPrice");
+                LocalDateTime departureDate = rs.getTimestamp("departureDate").toLocalDateTime();
+                LocalDateTime arrivalDate = rs.getTimestamp("arrivalDate").toLocalDateTime();
+                Long plane_id = rs.getLong("plane_id");
+                Plane plane = planeDAO.getById(plane_id);
+                Boolean isDeleted = rs.getBoolean("isDeleted");
+                if (!isDeleted)
+                    flights.add(Flight.builder().id(id).initialPrice(price).plane(plane).
+                            departureDate(departureDate).arrivalDate(arrivalDate).build());
```

Comment:
please check how your teammates used builder (code style)

Reply:
and you can add fields to builder without above lines

Reply:
changed to teammates code style